src/edu/macalester/comp127/hw0/Area.java

```diff
@@ -21,7 +23,8 @@ public static double getSquareArea(double sideLength) {
      */
     public static double getCircleArea(double radius) {
         // TODO: replace me with the correct formula
-        return radius * 1.0;
+        double area = 3.1415926535 * radius * radius;

```

Comment:
There's a Math class that contains a constant variable Pi. You can use `Math.PI` instead.